mod-inventory/src/test/java/org/folio/ModInventoryTests.java

```diff
@@ -31,4 +30,9 @@ public void tearDown() {
   void mod_inventoryTest() {
     runFeatureTest("inventoryFeatureTest");
   }
+
+  @Test
+  void mod_inventoryConsortiaTest() {
```

Comment:
@RomanChernetskyi  As far as I understand, update-ownership.feature will be triggered not from Java-class(here), it will be triggered from the "consortia-inventory-junit.feature"? This is because of specific consortia-mechanism setup. Am I right ?

Reply:
Yeap, consortia-inventory-junit.feature executes following steps:
- set up consortia env with 3 tenants
- execute tests
- destroy env